src/components/views/EditProfile.js

```diff
@@ -57,17 +57,17 @@ const FormField1 = props => {
     );
   }; */
 
-const EditProfile = () => {
+const EditProfile = (props) => {
     const history = useHistory();
 
-    const location = useLocation();
-    var myuser = location.state;
+    // const location = useLocation();
+    var myuser = props.currentUser;
 
     //const [birthday, setBirthday] = useState(null);
-    let [username, setUsername] = useState(null);
-    let [password, setPassword] = useState(null);
+    let [username, setUsername] = useState('');
```

Comment:
because of this change you arent able to change only one thing eg only the username because it checks if it is null on line 192. But im not sure if thats the reason

Reply:
I don't know, maybe you are right